### DISCUSSION TITLE
Marketing Team Weekly Meeting Planning 2022-03-16

### DISCUSSION THREAD

**Discussion by murillol** [WordPress/Marketing-Team#19] [Ideas]
Let's discuss!
@nalininsbs @eidolonnight

**Comment by murillol** [comment]
**Agenda Draft** 

Join the team at 15:00 UTC in the [Marketing channel on the Make WordPress Slack](https://wordpress.slack.com/archives/C0GKJ7TFA). Async contribution is welcome too.

1. Welcome
2. Getting to know each other
    a. Attendees
    b. Social question of the week
3. [Notes from the Last Meeting](https://docs.google.com/document/d/1S9JOiW1qICfz8oJ3tpfWdVohh1qsQg9G_J8GmXnmbX4/edit?pli=1#)
    a. Volunteer call for notetakers and for our bi-weekly coffee break.
4. Project Updates
    a. Month in WordPress
    b. Social Media Update
    c. GitHub updates and links
    d. WCEU Contributor Day
5. Diary/Calendar Dates
    a. Meeting time
    b. Follow this Google [Calendar with meetings, collaborations, and social activities](https://calendar.google.com/calendar/b/2?cid=NDQyN2g1cnZrN2EwYXA5Y3VpNDI5OGYzY3NAZ3JvdXAuY2FsZW5kYXIuZ29vZ2xlLmNvbQ). We also list important dates in this [Forthcoming Diary Dates](https://make.wordpress.org/marketing/2021/01/13/forthcoming-diary-dates/) post.
6. Open Floor

If you have any additions in mind, please comment below.

**Comment by nalininsbs** [comment]
Quick link to the Script document created by Nicholas last week https://docs.google.com/document/d/1HPVgQDtKReOnAvbyHS8jlnGS1gVNccYsfOfNifdAA7w/edit

Brett McSherry is doing notes
